.github/CONTRIBUTING.md

```
@@ -0,0 +1,54 @@
+./gradlew spotlessApply  # auto-format
+./gradlew ktlintCheck    # lint check
+```
+
+## Project conventions
+- **One issue → one small, vertically-sliced PR.** Each change should be independently shippable with its own tests. If a PR is growing large, split it.
+- **Branch naming:** `type/issue-<n>/short-desc` — e.g. `feat/issue-12/translatable-component`.
+- **Tests are required** for behavioural changes. We use [Kotest](https://kotest.io/); the `test` module provides component matchers — dogfood them.
+- **Formatting** is enforced by ktlint + Spotless. Run `./gradlew spotlessApply` before pushing.
+- **PR templates:** pick the template that matches your change (feature / bugfix / docs / chore). Link the issue with `Closes #<n>`.
+
+## Commit & PR title format (enforced)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)